.changeset/green-dancers-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@evervault/sdk': patch
+---
+
+We were duplicating information from our documentation on our website into the README. This becomes time consuming to maintain. Instead we now have a single source of truth
```

Comment:
I'd remove this `This becomes time consuming to maintain.` since our logs are public facing